tests/test_bookmarks.py

```diff
@@ -269,7 +274,7 @@ def test_run(self):
                 for primary_key in primary_keys:
 
                     # Verify that the inserted records are replicated by the 2nd sync and match our expectations
-                    schema_keys = set(self.expected_schema_keys(stream))
+                    set(self.expected_schema_keys(stream))
```

Comment:
I am not sure what this is doing but I don't think we need it.

Reply:
Ok I'll remove it.